certifiers/checkpoint.go

```diff
@@ -0,0 +1,101 @@
+package certifiers
+
+import (
+	"bytes"
+
+	"github.com/pkg/errors"
+
+	rtypes "github.com/tendermint/tendermint/rpc/core/types"
```

Comment:
Yes, please do this. I was trying to avoid changes to tendermint core, but now that it is merging, please do these kind of renames. I trust you to consolodate better.

Reply:
BlockLight? That name doesn't sit well with me.

What about SignedHeader? 

I could add that to types.go, don't want to change the rpc json output unless necesary...